### PR TITLE
Set test in parallel to false for Service Bus tests

### PR DESCRIPTION
**Pull request by JoshLove-msft**
This should help avoid flakiness in tests that depend on a singleton diagnostic listener.

### CODE REVIEW

File: sdk/servicebus/service.projects [hunk 1]
```diff
@@ -21,6 +21,9 @@
   <ItemGroup Condition="'$(SDKType)' == 'client'">
     <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Azure.Messaging.*\**\*.csproj" />
     <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Microsoft.Azure.WebJobs.Extensions.ServiceBus\**\*.csproj" />
+    <ProjectReference Update="$(MSBuildThisFileDirectory)Azure.Messaging.*\tests\*.csproj">
```
Comment (jsquire) [inline sdk/servicebus/service.projects:24]: Can you help me understand what the effect of this is?

Comment (JoshLove-msft) [reply sdk/servicebus/service.projects:24]: This is essentially saying that the Azure.Messaging.ServiceBus tests cannot be run in parallel with other test assemblies.

**Review by jsquire — APPROVED** [review]

File: sdk/servicebus/service.projects (hunk 1 above)
Comment (JoshLove-msft) [reply sdk/servicebus/service.projects:24]: @heaths discovered this recently and added to the docs - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/README.md#running-live-tests-serially

**Review by heaths — APPROVED** [review]

**Pull request merged**